src/fe-small-advertisements/src/index.tsx

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import { composeWithDevTools } from 'redux-devtools-extension';
 
-import App from './App';
+import App from './components/App';
+import { rootReducer } from './reducers';
 import reportWebVitals from './reportWebVitals';
 
+const store = createStore(rootReducer, composeWithDevTools());
```

Comment:
You need ti find a way to add this middleware to the store only when we are on `localhost`

Reply:
done